### ISSUE TITLE
Module load error due to "*" version dependency

### ISSUE THREAD

**Issue by woutgg** [paulcbetts/debug-electron#1]
In a project using electron-prebuilt-compile I got this error:
```
Uncaught Error: Cannot find module 'debug/browser'
```
which only occured on a newly set-up copy of my project. I found out this is due to moved source files in the current debug module (2.5.1).
I noticed the dependency version has been fixed in commit 7af59aafc4185ff9d70635e33378d24de9e97aa5, so a patch release to npm would probably solve this issue.

**Comment by woutgg** [comment]
Apologies, I noticed there _is_ a release which includes the fix.
For me, the problem lies with `electron-compile`, which depends on an older version of this module ([this line](https://github.com/electron/electron-compile/blob/master/package.json#L36)).
Should I file a new issue there?

While searching through my dependency tree, I noticed that same dependency in the `electron-compilers` and `spawn-rx` modules.
Just thought to mention that as those modules belong to the same account and might cause the same issue as this one.
